src/editor/dialog/error_dialog.cpp

```diff
@@ -0,0 +1,35 @@
+
+#ifdef TOOLS_ENABLED
+
+#include <editor/editor_settings.h>
+#include <scene/gui/texture_rect.h>
+#include <scene/gui/rich_text_label.h>
+#include <editor/editor_scale.h>
+#include "error_dialog.h"
+
+ErrorDialog::ErrorDialog(): error_message_label(memnew(RichTextLabel)) {
```

Comment:
I think it misses call to super class constructor.

Reply:
Hmm i blindly copied it from the About dialog which in turn i copied from the C# one. Didn't check that. I'll look into if it's necessary or not